seaborn_objects_recipes/recipes/plotting.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+from matplotlib import style
+from dataclasses import dataclass
+import statsmodels.formula.api as smf
+import seaborn.objects as so  
+
+@dataclass
+class RegressionWithCI(so.PolyFit):
```

Comment:
OK, great. So I think there are two options for this naming wise: 

- call it `PolyFit`, and just add the new argument for `ci=...`, or call it `PolyFitCI` so it's clear what it draws from and that it will support all of PolyFits options (like `order` and `gridsize`).
- Docs can also reference that it accepts everything for PolyFit and leave a link, like many plotting functions do with reference to matplotlib options. e.g. https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.hist.html